components/checkbox/index.css

```diff
@@ -152,395 +167,251 @@
 			}
-		}
 
-		&:hover {
-			.spectrum-Checkbox-input:checked + .spectrum-Checkbox-box::before,
-			.spectrum-Checkbox-box::before {
-				border-color: var(--highcontrast-checkbox-highlight-color-hover, var(--mod-checkbox-control-selected-color-hover, var(--spectrum-checkbox-control-selected-color-hover)));
+			&::after {
+				forced-color-adjust: none;
+				box-shadow: 0 0 0 var(--mod-checkbox-focus-indicator-thickness, var(--spectrum-checkbox-focus-indicator-thickness)) var(--highcontrast-checkbox-focus-indicator-color, var(--mod-checkbox-focus-indicator-color, var(--spectrum-checkbox-focus-indicator-color)));
+				margin: calc(var(--mod-checkbox-focus-indicator-gap, var(--spectrum-checkbox-focus-indicator-gap)) * -1);
 			}
 		}
-	}
 
-	/* Invalid Indeterminate */
-	&.is-invalid.is-indeterminate {
-		.spectrum-Checkbox-box,
-		.spectrum-Checkbox-input:checked + .spectrum-Checkbox-box {
-			&::before {
-				border-color: var(--highcontrast-checkbox-color-default, var(--mod-checkbox-invalid-color-default, var(--spectrum-checkbox-invalid-color-default)));
-				border-width: var(--mod-checkbox-selected-border-width, var(--spectrum-checkbox-selected-border-width));
-			}
+		&:focus-visible + .spectrum-Checkbox-label {
+			color: var(--highcontrast-checkbox-content-color-focus, var(--mod-checkbox-content-color-focus, var(--spectrum-checkbox-content-color-focus)));
 		}
+	}
 
-		&:hover {
-			.spectrum-Checkbox-box::before,
-			.spectrum-Checkbox-input:checked + .spectrum-Checkbox-box::before {
-				border-color: var(--highcontrast-checkbox-color-default, var(--mod-checkbox-invalid-color-hover, var(--spectrum-checkbox-invalid-color-hover)));
-			}
+	.spectrum-Checkbox-box {
+		display: flex;
+		align-items: center;
+		justify-content: center;
 
-			.spectrum-Checkbox-label {
-				color: var(--highcontrast-checkbox-content-color-hover, var(--mod-checkbox-content-color-hover, var(--spectrum-checkbox-content-color-hover)));
-			}
-		}
-	}
-}
+		position: relative;
+		box-sizing: border-box;
+		inline-size: var(--mod-checkbox-control-size, var(--spectrum-checkbox-control-size));
+		block-size: var(--mod-checkbox-control-size, var(--spectrum-checkbox-control-size));
+
+		/* Fix vertical alignment when not wrapping since we're flex-start */
+		margin: var(--mod-checkbox-margin-block, var(--spectrum-checkbox-top-to-control)) 0;
+		flex-grow: 0;
+		flex-shrink: 0;
 
-/* Emphasized */
-.spectrum-Checkbox--emphasized {
-	/* Checked and Indeterminate Default States */
-	.spectrum-Checkbox-input:checked + .spectrum-Checkbox-box,
-	&.is-indeterminate .spectrum-Checkbox-input:checked + .spectrum-Checkbox-box,
-	&.is-indeterminate .spectrum-Checkbox-box {
 		&::before {
-			border-color: var(--highcontrast-checkbox-highlight-color-default, var(--mod-checkbox-emphasized-color-default, var(--spectrum-checkbox-emphasized-color-default)));
-		}
-	}
+			forced-color-adjust: none;
```

Comment:
Ideally all uses of `forced-color-adjust` should be organized within the forced-colors media query. I think that makes it a lot easier to reason about the forced-color styles.